projectcard/io.py

```diff
@@ -186,7 +186,7 @@ def read_cards(
     filepath: Union[Collection[str], str],
     filter_tags: Collection[str] = [],
     recursive: bool = False,
-    _cards: Mapping[str, ProjectCard] = {},
+    _cards: Mapping[str, ProjectCard] = None,
```

Comment:
Yeah..cause the notebook will create scenario from a prior scenario. The _cards from prior call will be preserved in the next call. If we don't initialized it, it will generate errors saying some projects are already there.
I found the same issue in network wrangler project_card_list.

Reply:
> The _cards from prior call will be preserved in the next call. 

That is bizarre behavior, but if this fixes it - awesome.

Reply:
I think it is because we make changes to mutable objects, like dictionary and list. It can affect the original objects when they are passed around